src/test/javascript/spec/component/text-exercise/example-text-submission.component.spec.ts

```diff
@@ -238,12 +238,12 @@ describe('ExampleTextSubmissionComponent', () => {
         // THEN
         expect(comp.state.constructor.name).toEqual('EditState');
         expect(assessmentsService.deleteExampleAssessment).toHaveBeenCalledWith(EXERCISE_ID, EXAMPLE_SUBMISSION_ID);
-        expect(comp.submission?.blocks).toBeUndefined();
-        expect(comp.submission?.results).toBeUndefined();
-        expect(comp.submission?.latestResult).toBeUndefined();
-        expect(comp.result).toBeUndefined();
-        expect(comp.textBlockRefs).toHaveLength(0);
-        expect(comp.unusedTextBlockRefs).toHaveLength(0);
+        expect(comp.submission?.blocks).toBe(undefined);
+        expect(comp.submission?.results).toBe(undefined);
+        expect(comp.submission?.latestResult).toBe(undefined);
+        expect(comp.result).toBe(undefined);
+        expect(comp.textBlockRefs.length).toBe(0);
```

Comment:
Oh no what do you have against `toHaveLength()`? Can we undo these changes again? 😅